mozilla_django_oidc_db/models.py

```diff
@@ -176,6 +176,25 @@ class OpenIDConnectConfig(CachingMixin, SingletonModel):
         blank=True,
     )
 
+    @property
+    def oidc_rp_scopes(self):
```

Comment:
```suggestion
    def oidc_rp_scopes(self) -> str:
```
type hints are cool :tm: